src/pages/essentials-for-yoothemepro/forms/elements/fields.md

```diff
@@ -235,35 +235,152 @@ Month input relies on the browser native implementation, the picker and icon sty
-| **Min** | Defines a week that the input value must constrain to, `greater than` or `equal to`. |
-| **Max** | Defines a week that the input value must constrain to, `lower than` or `equal to`. |
+| Validation | Description | Dynamic |
+| ---------- | ----------- | :-----: |
+| **Min** | Defines a week that the input value must constrain to, `greater than` or `equal to`. | &#x2713; |
+| **Max** | Defines a week that the input value must constrain to, `lower than` or `equal to`. | &#x2713; |
 
 {% callout type="warning" title="Styling Limitations" %}
-Week input relies on the browser native implementation, the picker and icon style is unconsistent and it customization not possible.
+Week input relies on the browser's native implementation, the picker and icon style is inconsistent and it customization is not possible.
 {% /callout %}
 
 ---
 
+## Hidden Element
+
+The Hidden element displays a standard `<input type="hidden">` field. It inherits the following common settings from Input, `Control Name` and `Value`. No other settings are available.
+
+---
+
 ## Textarea Element
 
-The Textarea element Displays a standard `<textarea>` HTML element.
+The Textarea element displays a standard `<textarea>` HTML element.
+
+| Setting | Description | Dynamic |
+| ------- | ----------- | :-----: |
+| **Placeholder** | Defines a text to be shown when there is no input value yet. | &#x2713; |
+| **Width** | Defines the field width by applying a [`uk-form-width-{width}`](https://getuikit.com/docs/form#width-modifiers) style. |
+| **Rows** | Defines the number of visible text lines by setting a value for `rows` attribute. |
+
+| Validation | Description | Dynamic |
+| ---------- | ----------- | :-----: |
+| **Min length** | Defines a content length that the textarea value must constrain to, `greater than` or `equal to`. | &#x2713; |
+| **Max length** | Defines a content length that the textarea value must constrain to, `lower than` or `equal to`. | &#x2713; |
+| **Pattern** | Defines a [regular expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions) pattern that the input value must match to. | &#x2713; |
+
+---
+
+## Upload Element
+
+The Upload element displays by default a `<input type="file">` HTML element and a button with text.
+
+| Setting | Description | Dynamic |
+| ------- | ----------- | :-----: |
+| **Upload to** | Defines a path where the file will be uploaded. | &#x2713; |
+| **Avoid filename collisions** | Defines if the filename should change on upload, if conflicting with existing files. The newly created filename will be the original name appended with `_{n}`, being `n` a secuancial number. |
+| **Allow multiple uploads** | Defines if multiple files can be selected for upload, by adding the attribute `multiple` to `<input type="file">`. |
+| **Filename** | Defines an optional name to be set to the the file, upon upload. |
+
+| Validation | Description | Dynamic |
+| ---------- | ----------- | :-----: |
```

Comment:
Any reason for using `:-----:` instead of just `-----` ?

Reply:
It centers the content, in this case the check icon.